opt/cs50/bin/flask

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+# If $FLASK_APP isn't set, check for application.py (for backwards compatibility), else default to app.py
+if [[ -z "$FLASK_APP" ]]; then
+    if [[ -f "application.py" ]]; then
+        FLASK_APP="application.py"
+    else
+        FLASK_APP="app.py"
+    fi
+fi
+
+# Disable debugger by default if $FLASK_ENV not explicitly set to "development"
+# https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features
+if [[ "$FLASK_ENV" != "development" ]]; then
+    FLASK_DEBUG="${FLASK_DEBUG:-0}"
```

Comment:
Is debug mode not disabled by default?

Reply:
I think this was originally meant to be super-explicit that we were disabling debug mode by default, but I removed.